static/app/views/dashboardsV2/data.tsx

```diff
@@ -2,8 +2,10 @@ import {t} from 'app/locale';
 
 import {DashboardDetails} from './types';
 
+export const emptyDashboardId = 'default-overview';
+
 export const EMPTY_DASHBOARD: DashboardDetails = {
-  id: '',
+  id: emptyDashboardId,
```

Comment:
I think this should be an empty string. `default-overview` is a reserved dashboard ID.

Reply:
maybe the name here is not quire correct, but according to below logic I think this should be the default dashboard ID

https://github.com/getsentry/sentry/blob/9daf9246c86dbd43c8f3d1e456a81ef0dea31de8/static/app/views/dashboardsV2/orgDashboards.tsx#L92

Reply:
When creating a dashboard I pass an empty id https://github.com/getsentry/sentry/blob/d63dec406c78e4245999b41a456f97084507a4fb/static/app/views/dashboardsV2/detail.tsx#L221